.github/workflows/hrm-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: hrm-ci
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
Great.  Side note -- I've just upgraded our AWS staging env to Node 12.16.1.  It had been on Node 10.  If that all goes well we can also upgrade production.